tests/unit/l3_dump.py-test.c

```diff
@@ -38,6 +51,21 @@ main(const int argc, const char **argv)
     l3_log_simple("Potential memory overwrite (addr, size)", 0xdeadbabe, 1024);
     l3_log_simple("Invalid buffer handle (addr)", 0xbeefabcd, 0);
 
-    return 0;
+    printf("Generated slow log-entries to log-file: %s\n", log);
 }
 
+void test_l3_fast_log(void)
+{
+    const char *log = "/tmp/l3.c-fast-unit-test.dat";
+    int e = l3_init(log);
+    if (e) {
+        abort();
+    }
+    l3_log_fast("Fast-log-msg: Args(1,2)", 1, 2);
+    l3_log_fast("Fast-log-msg: Args(3,4)", 3, 4);
+    l3_log_fast("Fast-log-msg: Args(10,20)", 10, 20);
```

Comment:
@gregthelaw - There is still something going slightly wrong with fast-logging, even when LOC is not enabled.

Unit-tests, by default, run without LOC. 

In [this test output](https://github.com/undoio/l3/actions/runs/8591118511/job/23539502005?pr=16), for the case of `Build-and-Run-Unit-tests`, the value printed for `arg2` for the call on L64 is incorrect:

```
python3 l3_dump.py /tmp/l3.c-fast-unit-test.dat ./build/release/bin/unit/l3_dump.py-test
tid=3102 'Fast-log-msg: Args(1,2)' arg1=1 arg2=93990550136310
tid=3102 'Fast-log-msg: Args(3,4)' arg1=3 arg2=4
tid=3102 'Fast-log-msg: Args(10,20)' arg1=10 arg2=20
```

I added L66, to see if other values are printed correctly; they are.

Strangely, it seems like the very 1st fast-log call's 2nd argument is always being clobbered.

I don't think it's an issue with the Py script that plucks out these values.

I suspect the assembly is stashing something wrong for the very 1st log-entry.

Can you please take a look at this small-item ... whenever you get a chance?

Reply:
This whole cross-checking needs pytests, and I will build some once this core piece gets integrated. (That's a bit of new test infra that needs to be implemented from scratch.)

Reply:
OK, I figured out the stupid assembly bug!

Reply:
Applied this fix and verified that this bug is now fixed.